recipes/platon/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set version = "1.0.0" %}
+{% set sha256 = "5b04a9774d3bf90d6acb1a58a943d9cd6e088ba48bee6965198533105835bbf9" %}
+
+package:
+  name: platon
+  version: '{{ version }}'
+
+source:
+  url: https://github.com/oschwengers/platon/archive/v{{ version }}.tar.gz
+  sha256: '{{ sha256 }}'
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - platon = platon.platon:main
+
+requirements:
+  host:
+    - python >=3
+
+  build:
```

Comment:
build should be moved up and host should be the second section.
Also pip should be in host not in build.

tar is not needed anymore?

Reply:
Ok, I wasn't aware of this. 

Tar is still needed as it's the source of the local pip installation. This way, it merely depends on the released GitHub version but not on a published PyPi.org-release.